buying-market/src/main/java/buyingMarket/services/RecieptService.java

```diff
@@ -0,0 +1,29 @@
+package buyingMarket.services;
+
+import buyingMarket.model.Reciept;
+import buyingMarket.model.Transaction;
+import buyingMarket.repositories.TransactionRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+public class RecieptService {
+    public final TransactionRepository transactionRepository;
+
+    @Autowired
+    public RecieptService(TransactionRepository transactionRepository) {
+        this.transactionRepository = transactionRepository;
+    }
+
+    public Reciept getReciept(long userId){
+        List<Transaction> userTransactions = transactionRepository.findAllByUser(userId);
```

Comment:
Ovo mi izgleda kao da Reciept ce stalo da sadrzi sve Transakcije koje je korisnik imao ikada ( sto i nema smisla ), treba da se doda polje u Reciept i Transaction koje ih jednistveno pomaze npr da se doda RecieptId i u Transaction da se doda to kao polje

Reply:
U sustini one to many veza da bude?

Reply:
Mislim Reciept treba da ima zapravo istoriju svih transakcija koliko sam ja razumeo.